SPECS/helm/helm.spec

```diff
@@ -55,8 +56,11 @@ install -m 755 ./helm %{buildroot}%{_bindir}
 go test -v ./cmd/helm
 
 %changelog
+* Wed May 29 2024 Neha Agarwal <nehaagarwal@microsoft.com> - 3.13.2-3
```

Comment:
We missed to bump the actual `Release` tag.

Reply:
Thanks, fixed and rebuilt successfully